auth-backends/src/main/elm/sources/AuthBackends.elm

```diff
@@ -428,15 +417,8 @@ getErrorMessage e =
     let
         errMessage =
             case e of
-                Http.BadStatus b ->
```

Comment:
This is a regression, but getting the request content extends my current elm skill.

Reply:
Ok, `Http.Detailed`